src/routes/retro-certs.js

```diff
@@ -9,6 +9,39 @@ const weeksCompleted = require("../utils/checkFormData");
 function createRouter() {
   const router = Router();
 
+  async function staffViewAuthStatus(postJson, responseJson) {
+    responseJson.status = AUTH_STRINGS.statusCode.userNotFound;
+    const [userRecord] = await Promise.all([
+      cosmos.getUserByNameDobSsn(
+        postJson.lastName || "",
+        postJson.dob,
+        postJson.ssn
+      ),
+    ]);
+    if (!userRecord) {
+      console.log("failed staff view login");
+      return responseJson;
+    }
+
+    console.log("staff view login", userRecord.id);
+    userRecord.lastStaffLogin = Date.now();
```

Comment:
extremely minor nit, how about staffViewedAt? since the staff person isn't logging in as the user, can't edit anything, they're just viewing their form

Reply:
Makes sense. How about staffLastViewedAt?